src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -525,5 +526,41 @@ void SubgraphBaseTest::init_input_shapes(const std::vector<InputShape>& shapes)
         }
     }
 }
+
+ElementType SubgraphBaseTest::get_default_imp_precision_type(ElementType type) {
+#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
+    return type;
+#endif
+#if defined(OPENVINO_ARCH_RISCV64)
+    return type;
+#endif
+#if defined(OPENVINO_ARCH_X86_64)
+    const std::string key = ov::hint::inference_precision.name();
+    const std::string KEY_ENFORCE_BF16 = "ENFORCE_BF16";
+    // if is not float
+    if (type != ElementType::f16 && type != ElementType::f32 && type != ElementType::bf16) {
```

Comment:
Why such cpu specific is placed in shared lib?
plese move this function logic to `core_configuration function`

We have to avoid a device specific in shared lib

Reply:
already move the function into cpu plugin test code